src/Microsoft.DotNet.Build.CloudTestTasks/Microsoft.DotNet.Build.CloudTestTasks.csproj

```diff
@@ -26,22 +26,7 @@
     <Compile Include="$(CommonPath)\ZipFileCreateFromDirectory.cs">
       <Link>ZipFileCreateFromDirectory.cs</Link>
     </Compile>
-    <Compile Include="SendJobsToHelix.cs" />
-    <Compile Include="SimulateHelixExecution.cs" />
-    <Compile Include="ValidateWorkItems.cs" />
-    <Compile Include="AzureConnectionStringBuildTask.cs" />
-    <Compile Include="AzureHelper.cs" />
-    <Compile Include="CreateAzureContainer.cs" />
-    <Compile Include="DownloadFromAzure.cs" />
-    <Compile Include="FilterForUpload.cs" />
-    <Compile Include="GetPerfTestAssemblies.cs" />
-    <Compile Include="ListAzureContainers.cs" />
-    <Compile Include="SendToHelix.cs" />
-    <Compile Include="UploadClient.cs" />
-    <Compile Include="UploadToAzure.cs" />
-    <Compile Include="WriteBuildStatsJson.cs" />
-    <Compile Include="RemoveItemMetadata.cs" />
-    <Compile Include="WriteItemsToJson.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)*.cs" />
```

Comment:
But why? I realize wildcards seem like a good idea but they usually do nothing but cause trouble in some way.